cli/app/ui_layout_rendering_helpers.go

```
@@ -82,26 +86,60 @@ func wrappedEditableInputLines(width int, spec uiEditableInputRenderSpec) []stri
+	}
+	runes := []rune(line)
+	displayCol := 0
+	for index, r := range runes {
+		rw := runewidth.RuneWidth(r)
+		if rw < 1 {
+			rw = 1
+		}
+		if cursorCol < displayCol+rw {
+			prefix := string(runes[:index])
+			suffix := string(runes[index+1:])
+			return lineStyle.Render(prefix) + lineStyle.Copy().Reverse(true).Render(string(r)) + lineStyle.Render(padANSIRight(suffix, width-displayCol-rw))
```

golangci-lint (2.11.4)

[error] SA1019: lineStyle.Copy is deprecated: to copy just use assignment (i.e. a := b). All methods also return a new style. 

(staticcheck)